app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
```

Comment:
You're splitting the destination path into its components. However, you don't need to do this manually, as Python's `os.path` module provides functions to work with paths. You can use `os.path.split()` to get the head and the tail of the path, where the tail will be the last pathname component and the head will be everything leading up to that.